packages/javascript/FlatfileJavascript.ts

```diff
@@ -271,13 +247,109 @@ function initializeIFrameConfirmationModal(
   }
 }
 
+type InitSpaceType = ISpace & {
+  isAutoConfig: boolean
```

Comment:
Why `isAutoConfig` option instead of just using the `autoConfigure` name?

Reply:
I added a separate property because the property is _derived_ from conditions - the original `ISpace` encapsulates the properties that are _passed into_ the embed. I thought it made sense to have something separate so not to confuse with the property that is set via config vs. property derived